scripts/_common.sh

```diff
@@ -1,200 +1 @@
-	if [ ! -e "$node_version_path/$nodejs_version" ]
-	then
-		ln --symbolic --force --no-target-directory $node_version_path/$real_nodejs_version $node_version_path/$nodejs_version
-	fi
-
-	# Store the ID of this app and the version of node requested for it
-	echo "$YNH_APP_ID:$nodejs_version" | tee --append "$n_install_dir/ynh_app_version"
-
-	# Store nodejs_version into the config of this app
-	ynh_app_setting_set $app nodejs_version $nodejs_version
-
-	# Build the update script and set the cronjob
-	ynh_cron_upgrade_node
-
-	ynh_use_nodejs
-}
-
-# Remove the version of node used by the app.
-#
-# This helper will check if another app uses the same version of node,
-# if not, this version of node will be removed.
-# If no other app uses node, n will be also removed.
-#
-# usage: ynh_remove_nodejs
-ynh_remove_nodejs () {
-	nodejs_version=$(ynh_app_setting_get $app nodejs_version)
-
-	# Remove the line for this app
-	sed --in-place "/$YNH_APP_ID:$nodejs_version/d" "$n_install_dir/ynh_app_version"
-
-	# If no other app uses this version of nodejs, remove it.
-	if ! grep --quiet "$nodejs_version" "$n_install_dir/ynh_app_version"
-	then
-		$n_install_dir/bin/n rm $nodejs_version
-	fi
-
-	# If no other app uses n, remove n
-	if [ ! -s "$n_install_dir/ynh_app_version" ]
-	then
-		ynh_secure_remove "$n_install_dir"
-		ynh_secure_remove "/usr/local/n"
-	fi
-}
-
-# Set a cron design to update your node versions
-#
-# [internal]
-#
-# This cron will check and update all minor node versions used by your apps.
-#
-# usage: ynh_cron_upgrade_node
-ynh_cron_upgrade_node () {
-	# Build the update script
-	cat > "$n_install_dir/node_update.sh" << EOF
-#!/bin/bash
-
-version_path="$node_version_path"
-n_install_dir="$n_install_dir"
-
-# Log the date
-date
-
-# List all real installed version of node
-all_real_version="\$(find \$version_path/* -maxdepth 0 -type d | sed "s@\$version_path/@@g")"
-
-# Keep only the major version number of each line
-all_real_version=\$(echo "\$all_real_version" | sed 's/\..*\$//')
-
-# Remove double entries
-all_real_version=\$(echo "\$all_real_version" | sort --unique)
-
-# Read each major version
-while read version
-do
-	echo "Update of the version \$version"
-	sudo \$n_install_dir/bin/n \$version
-
-	# Find the last "real" version for this major version of node.
-	real_nodejs_version=\$(find \$version_path/\$version* -maxdepth 0 | sort --version-sort | tail --lines=1)
-	real_nodejs_version=\$(basename \$real_nodejs_version)
-
-	# Update the symbolic link for this version
-	sudo ln --symbolic --force --no-target-directory \$version_path/\$real_nodejs_version \$version_path/\$version
-done <<< "\$(echo "\$all_real_version")"
-EOF
-
-	chmod +x "$n_install_dir/node_update.sh"
-
-	# Build the cronjob
-	cat > "/etc/cron.daily/node_update" << EOF
-#!/bin/bash
-
-$n_install_dir/node_update.sh >> $n_install_dir/node_update.log
-EOF
-
-	chmod +x "/etc/cron.daily/node_update"
-}
+#/bin/bash
```

Comment:
Uh isn't it simpler to just remove the file (and corresponding sourcings in the scripts)